webapp/components/CommentCard/CommentCard.vue

```diff
@@ -149,8 +148,9 @@ export default {
       return `#${this.anchor}` === anchor
     },
     reply() {
-      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
+      const message = { slug: this.comment.author.slug + ' ', id: this.comment.author.id }
       this.$emit('reply', message)
+      this.$toast.success(this.$t(`post.comment.addReply`))
```

Comment:
I really dislike this solution. You get a success message even before a comment has been saved.